tests/test_github.py

```diff
@@ -46,15 +48,15 @@ def test_fetch_issues_with_events():
     # Mock events request
     responses.add(responses.GET, TEST_EVENTS_URL, json=expected_events, status=200)
 
-    # Assert that response with events has expected format
-    response_with_events = github.fetch_issues(TEST_URL, True)
     expected_with_events = expected
     expected_with_events[0]["events"] = expected_events
 
+    # Assert that response with events has expected format
+    response_with_events = github.fetch_issues(TEST_URL, True)
```

Comment:
Nit: why did you move this?

Reply:
I've separated building expected data from the actual response to make it a bit more readable

Reply:
Ok, WFM.